cmake/cpu_extension.cmake

```diff
@@ -16,10 +16,16 @@ include_directories("${CMAKE_SOURCE_DIR}/csrc")
 #
 # Check the compile flags
 #
+if (CMAKE_SYSTEM_PROCESSOR STREQUAL "ppc64le")
+    list(APPEND CXX_COMPILE_FLAGS
+    "-fopenmp"
+    "-DVLLM_CPU_EXTENSION")
+else()
 list(APPEND CXX_COMPILE_FLAGS
     "-fopenmp"
     "-mf16c"
     "-DVLLM_CPU_EXTENSION")
+endif()
```

Comment:
similar to this block - https://github.com/vllm-project/vllm/blob/284203f171d86a9581295436d6175246215437fd/cmake/cpu_extension.cmake#L57

Reply:
Above flag was applied for all architectures. We know for Power its not applicable so added exclusive check to avoid it for Power.
Adding exclusive check for x86 may avoid it applying for other supported architectures.

Reply:
x86 is the only architecture that supports this flag, so I don't see how it would hurt
Please see this incoming PR for ARM support for an illustration of what I'm asking for here 
https://github.com/vllm-project/vllm/pull/9957/files#diff-46b07f808e558aff84dcc365e53a91400c4be399757ff40c2da4278a50e89fbdL19-R24